index.html

```diff
@@ -79,7 +82,7 @@
 
                         <h1 class="brand-heading">Breeze '16</h1>
                         <p class="intro-text">Shiv Nadar University</p>
-                        <a href="#contact" class="btn btn-circle page-scroll">
```

Comment:
Should've left the link for Contact section too, and added another for About Us.
Contact section could have the link to our youtube, facebook and instagram pages.